README.md

```diff
@@ -36,21 +38,20 @@ The `GITHUB_TOKEN` secret is [automatically generated by GitHub](https://help.gi
 
 By default, you probably don’t need to configure `image-actions`. However, if you’d like to ignore specific file paths, or change image compression options, read on.
 
-Change the configuration options by adding a `.github/calibre/image-actions.yml` file:
+Change the configuration options by adding arguments to the action:
 
 ```yml
-jpeg:
-  quality: 80
-png:
-  quality: 80
-webp:
-  quality: 80
-ignorePaths:
-  - "node_modules/**"
+with:
+  githubToken: ${{ secrets.GITHUB_TOKEN }}
+  jpegQuality: 80
+  pngQuality: 80
+  webpQuality: 80

```

Comment:
This new stanza doesn't make it possible to send other options to the jpeg, png or webp processors. Have you thought about how that might be possible?

Reply:
I'm not a fan of changing this to hardcoded args. What we have at the moment is more flexible. Is there a middle-ground?

Reply:
currently it's still possible to use the old method, this would just become the "recommended" method, but it's in no way required.

Reply:
@chabad360 [Here is a bit more background other configuration options](https://github.com/calibreapp/image-actions/issues/7#issuecomment-541465444) that would be useful to consider, would you mind taking a look?

Reply:
I'd say that these kinds of things are arbitrary, if you want to add another type of argument, add it to the `action.yml` and create a handler for it and you're good!

It doesn't really make a difference, but it will be a lot easier to do stuff like this once GitHub Actions adds support for arrays. But until then, we can just say that for advanced configuration you'll need to specify `configPath: ".github/calibre/config.yml"` (or somthing like that, let's not make the file location an arbitrary one and leave it customizable), and create a yaml file with your configuration in that specified location, which contains all your advanced configs.

Reply:
> (or something like that, let's make the file location customizable)

I've added such a capability and added it in the [README](https://github.com/calibreapp/image-actions/pull/29/files#diff-04c6e90faac2675aa89e2176d2eec7d8R56-R7) to avoid this seeming like it's no longer supported/working.